templates/types/streaming/nextjs/app/components/ui/file-uploader.tsx

```
@@ -50,30 +52,51 @@ export default function FileUploader({
+      const fileExtension = file.name.split(".").pop() || "";
+      const extensionFileError = checkExtension(fileExtension);
+      if (extensionFileError) {
+        onFileUploadError(extensionFileError);
+        continue;
+      }
+
+      if (isFileSizeExceeded(file)) {
+        onFileUploadError(
+          `File size exceeded. Limit is ${fileSizeLimit / 1024 / 1024} MB`,
+        );
+        continue;
```

Biome

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)